app/controllers/settings_controller.rb

```diff
@@ -45,8 +45,12 @@ def edit
       end
 
       settings.each do |name, value|
-        # remove blank values in array settings
-        value.delete_if(&:blank?) if value.is_a?(Array)
+        if value.is_a?(Array)
+          # remove blank values in array settings
+          value.delete_if(&:blank?)
+        else
+          value = value.strip
```

Comment:
I thought about doing this myself.
Do you think this is a safe operation?
I am not quite aware of all the settings we have (including plugins) and stripping whitespace on **all** of them sounds like a rough/risky operation.